openapi.yaml

```diff
@@ -6471,6 +6471,13 @@ components:
           uniqueItems: true
         labels:
           $ref: '#/components/schemas/Labels'
+        implementation_mode:
```

Comment:
This property [seems like it is in development](https://kongstrong.slack.com/archives/CAW1MAFLZ/p1753118261670139?thread_ts=1753116515.745819&cid=CAW1MAFLZ) as well, should we remove it using an overlay?